packages/builder-vite/README.md

```diff
@@ -104,6 +104,37 @@ module.exports = {
 };
 ```
 
+## TypeScript
+
+- Configure your `.storybook/main.ts` to use TypeScript
+
+```typescript
+import { StorybookViteConfig } from "@storybook/builder-vite"
+
+const config: StorybookViteConfig = {
```

Comment:
Doesn't this need to be default exported at the end?

Reply:
Oh, I mean just doing the basic TS usage... I'll add it

Reply:
Thanks.  I think sometimes people copy-paste examples and then can get confused when they don't work directly.  :)